config/settings/production.yml

```diff
@@ -138,3 +138,4 @@ member_services_secret: <%= ENV['MEMBER_SERVICES_SECRET'] %>
 oxr_app_id: <%= ENV['OXR_APP_ID'] %>
 api_key: <%= ENV['CHAMPAIGN_API_KEY'] %>
 mailer_topic_arn: <%= ENV['MAILING_TOPIC_ARN'] %>
+db_pool_size: <%= ENV['DPS'] %>
```

Comment:
`DPS` - a bit cryptic, but we're so low on ENV space.

Reply:
> but we're so low on env space
ouch